pkg/operator/controller.go

```diff
@@ -211,6 +212,29 @@ func (c *Controller) createOrUpdateResources(cr *imageregistryv1.Config) error {
 	return nil
 }
 
+// getRoutes returns a list of all routes configured for the image registry, including
+// the default route if configured.
+func (c *Controller) getRoutes(cr *imageregistryv1.Config) ([]*routev1.Route, error) {
+	var routes []*routev1.Route
+	if cr.Spec.DefaultRoute {
+		if route, err := c.listers.Routes.Get(defaults.RouteName); err != nil {
+			klog.V(5).Infof("unable to get default route: %s", err)
```

Comment:
V(5) won't be visible even if you enable Debug logging. Only Trace or TraceAll that also includes HTTP requests from client-go.

Reply:
done, ptal.